media-sound/cmus/cmus-2.7.1.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A Console Based Audio Player "
+DESCRIPTION="
+cmus (C* MUsic Player) is a small, fast and powerful console audio player which supports most major audio formats. Various features include gapless playback, ReplayGain support, MP3 and Ogg streaming, live filtering, instant startup, customizable key-bindings, and vi-style default key-bindings."
```

Comment:
This should be wrapped to 80 characters using \s, you made it longer than it was before...